pom.xml

```diff
@@ -637,7 +637,6 @@
         <guava.version>25.1-jre</guava.version>
 
         <jutf7.version>1.0.0</jutf7.version>
-        <plugin.mailetdocs.version>0.1</plugin.mailetdocs.version>
```

Comment:
is it specified somewhere?

Reply:
Not sure to get your question... What do you mean?

Reply:
We reference `mailetdocs-maven-plugin`, but where do we specify the version?

Reply:
Very good point, went a bit fast on that one... Didn't get that the version was defined in mailet pom with plugin management and that then every plugin invocation in submodule would inherit it. However `0.1`... `mailetdocs-maven-plugin` is a submodule too, so if we put `${project.version}` for the version, is that alright for you?

Reply:
yes, it is.

Reply:
Please recheck mailet-doc generation after that using the docker specified in the README

Reply:
Ok... Doesn't work! Let's put this on stand by until I (or someone else) finds what's wrong...